models/account_approval.py

```diff
@@ -1,9 +1,51 @@
 # -*- coding: utf-8 -*-
 
-from odoo import fields, models
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
 
 
 class AccountBankStatementLine(models.Model):
     _inherit = "account.bank.statement.line"
 
     approved_by = fields.Many2one('res.users', string='Approved by')
+
+
+class DiscountSettings(models.Model):
+    _name = 'discount.settings'
+    _description = 'Discount Settings'
+
+    name = fields.Char()
+    max_amount = fields.Integer(required=True)
+    user_ids = fields.One2many('res.users', 'discount_settings_id')
+    # TODO: Maybe you want a group_ids here to allow a whole group in once?
```

Comment:
@sbuhl

Reply:
@shortcut1337 Maybe more straightforward to allow a whole group in once, yes.

Reply:
As said in another comment,
Let's use the groups, that I still have to create: 
1. Junior
2. Mid-Level
3. Senior
4. Store Manager
5. Director